frontend/src/lib/components/LemonTable/TableRow.tsx

```diff
@@ -22,10 +22,12 @@ function TableRowRaw<T extends Record<string, any>>({
     onRow,
     expandable,
 }: TableRowProps<T>): JSX.Element {
-    const [isRowExpanded, setIsRowExpanded] = useState(false)
+    const [_isRowExpanded, setIsRowExpanded] = useState(false)
```

Comment:
The underscore is a bit sneaky, its meaning could be conveyed more explicitly with something like
`[isRowExpandedLocal, setIsRowExpandedLocal]`